samples/wrapper/wrapper.cpp

```diff
@@ -747,7 +747,10 @@ int TASK::run(int argct, char** argvt) {
         priority_val = 0;
     } else {
         if (aid.process_priority > 0) {
```

Comment:
Please also change this test to 

`if (aid.process_priority > CONFIG_PRIORITY_UNSPECIFIED) {`